storybook/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "storybook",
+  "version": "0.0.1",
```

Comment:
should it match the version of podman desktop ?

Reply:
website don't ?